yt/yt/server/query_tracker/handler_base.cpp

```diff
@@ -44,23 +44,63 @@ namespace NDetail {
 
 ///////////////////////////////////////////////////////////////////////////////
 
+TSharedRef TruncateRowset(TSharedRange<NTableClient::TUnversionedRow>& rows, size_t estimatedDataWeightLimit) {
+    if (estimatedDataWeightLimit < 1_MB) {
```

Comment:
Why is there such a big trash hold here? (1MB)

Reply:
If we reached 1MB, it means we tried 16mb, 8mb, 4mb, 2mb, 1mb, and chances of success for 512kb, 256kb, ... are small

Next revision I try 4 times: 16mb, 15mb, 1mb, 1 row